activerecord/lib/active_record/locking/optimistic.rb

```diff
@@ -47,6 +47,7 @@ module Locking
     #     self.locking_column = :lock_person
     #   end
     #
+    # Remember, optimistic locking will be ignored if you update locking column value.
```

Comment:
`Note that, optimistic locking will be ignored if you update the locking column value.`